src/config.ts

```diff
@@ -2,6 +2,8 @@ import { makeConfig } from 'cleaner-config'
 import { asNumber, asObject, asOptional, asString } from 'cleaners'
 
 export const asConfig = asObject({
+  searchTerm: asOptional(asString),
```

Comment:
This is a weird way to pass arguments to a script (normally we use the `process.argv` array), but it works, I guess.